run/models.py

```diff
@@ -0,0 +1,43 @@
+from django.db import models
+from storage.models import File
+from game_runner import settings
+import uuid
+
+
+# Initial scheme
+# FilePath
+#   +file : File
+#   +path : String
+#   +run : Run
+#
+# Run
+#   +token : Token
+#   +request_time : DateTime
+#   +start_time : DateTime
+#   +end_time : DateTime
+#   +input_file_paths : FilePath[]
+#   +output_file_paths : FilePath[]
+#   +log : File
+#   +game : Game
+#
+# API Methods:
+#   +run((game : Game, file_paths : FilePath[])[]) : tokens String[]
+#   +report(from_time : DateTime) : runs : Run[]
+
+class FilePath(models.Model):
+    file = File()
+    path = models.FilePathField()
```

Comment:
this probably should also be a foreign key to another model which is part of the Game design. Executing a run won't require you to specify the path of the file. But it simply asks you to give them some files. For example a client code. What you send will be something like this:
```
code_1: sjdjasdja
code_2 :jsdajsda
````
The path of these files are determined when specifying the game.